### ISSUE TITLE
I tried your examples, but only label displayed and not the texts or plot

### ISSUE THREAD

**Issue by ningxuca** [openanalytics/patientProfilesVis#4]
![Screenshot 2022-06-29 072515](https://user-images.githubusercontent.com/106090568/176461492-f51a493d-5ce6-489f-9720-a852ce2a4e13.png)

**Comment by ningxuca** [comment]
I noticed that you have linux system, I have window 10. I used your example code, I also downloaded pdfLatex, it seems to combine all individual pdf files into one file. Before downloading pdfLatex, the display of each pdf file under figures folder seem to be fine. After I downloaded from MikTEX, when I run the createSubjectProfileReport, a few pop up windows showed that some files are missing and asked if I would to download them. After I clicked OK, then the resulting patient profile pdf does not show the text or plot.

**Comment by lcougnaud** [comment]
Thanks for your interest in the package.

In your screenshot, the plots seem to be created but are empty.
Could you check:

1.  if the profile plot of a specific patient is correctly displayed in R? 
You can try: `print(dmPlots[[1]])` from this [example](https://github.com/openanalytics/patientProfilesVis/blob/master/patientProfilesVis/vignettes/patientProfilesVis-introduction.Rmd#L159) of the vignette.
2. if the profile plot is correctly exported to a pdf file?
The patient profiles are exported to pdf files with `cairo_pdf` (see [this line](https://github.com/openanalytics/patientProfilesVis/blob/master/patientProfilesVis/inst/template/subjectProfile.Rnw#L166)). 
Could you try to export one of the patient profile as:
```
cairo_pdf("patientProfiles.pdf")
print(dmPlots[[1]])
dev.off()
```

**Comment by ningxuca** [comment]
Hi Laure,

Thank you so much for getting back to me. I tried the code to just output
the 1st AEs to pdf. It worked correctly. see below.



[image: image.png]


Before, I used the following code to display, all the subjects had no graph
and the labels for the graphs were truncated on the left.

pathReport <- "subjectProfile.pdf"
createSubjectProfileReport(
  listPlots = list(
    dmPlots,
    mhPlots,
    aePlots,
    cmPlots
  ),
  outputFile = pathReport,
  shiny=FALSE,
  verbose=TRUE,
  exportFigures=TRUE
)
